packages/platform-sdk/test/integration-tests/sdk-v3/telemetry.test.ts

```diff
@@ -0,0 +1,88 @@
+import { ClientBuilder } from '@commercetools/ts-client'
+import {
+  //   type MiddlewareRequestLegacy,
+  //   type MiddlewareResponseLegacy,
+  //   createTelemetryMiddlewareV2,
+  MiddlewareRequest,
+  OTelemetryMiddlewareOptions,
+  Next,
+  datadog,
+  newrelic,
+} from '@commercetools/ts-sdk-apm'
+import { createApiBuilderFromCtpClient } from '@commercetools/platform-sdk'
+// import fetch from 'node-fetch'
```

Comment:
is there a reason why this is commented it out?

Reply:
Fetch is now natively included in nodejs since v18